recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - python
     - amostra <=1.0
     - attrs >=18.0
-    - bluesky <1.6.0rc
+    - bluesky ==1.6.0rc4
```

Comment:
Is this the right kind of pin to use?

Reply:
Let's put `>=` in case we want `rc5` to be available.